html/changelogs/aticius-totalorganfailure.yml

```diff
@@ -0,0 +1,10 @@
+ 
+author: Aticius
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+
+changes: 
+  - tweak: "Adjusted Liver's ability to heal toxins. Should be stronger now; but not too much.
+  - rscdel: "Removed anything outwardly dangerous from the Scrubber Clog event."
```

Comment:
next time, you don't need to delete the rest of the changelog stuff.

you might accidentally delete something important.

Reply:
Force of habit. My traditions will die with me.

or when i get fed up.

Reply:
travis is not happy

Reply:
One goat sacrifice later, Travis is happy.